packages/shared/src/libs/helpers/get-start-of-day/get-start-of-day.helper.ts

```diff
@@ -1,6 +1,6 @@
 import { startOfDay } from "date-fns";
 
-const getStartOfDay = (date: Date): Date => {
+const getStartOfDay = (date: Date | string): Date => {
```

Comment:
we should limit options in our custom helpers and use only one common format for payload, so we either need to pass date or string here. In this case it's better to pass date, not to care about date parsing in this helper. The same for getEndOfDay helper
```suggestion
const getStartOfDay = (date: Date): Date => {
```